magda-int-test/src/test/scala/au/csiro/data61/magda/test/util/MagdaMatchers.scala

```diff
@@ -26,6 +27,19 @@ object MagdaMatchers extends org.scalatest.Matchers {
     if (stemmer.stem) stemmer.getCurrent else string
   }
 
+  def lightEnglishStem(string: String): String = {
+    val stemmer = new LightEnglishStemmer
+    return stemmer.stem(string)
+  }
+
+  def stemString(term: String, useLightEnglishStemmer:Boolean = false) = {
```

Comment:
So do we still need the PorterStemmer? Can't we use the light english one for everything?

Reply:
The stemmer outcome is different. 
And `PorterStemmer` is part of standard built-in `english` analyzer
I was thinking a better way should be:
- we have a new function can read index definition 
- And construct the same analyzer process
- Then we can guarantee there is no exception --- at this moment, it's possible some combination of filters /tokenizerd results empty input as well